### PR TITLE
Added documentation for the Bazel-only workflow

### PR DESCRIPTION
**Pull request by UebelAndre**
This PR updates documentation for what users would need to do to be able to add `cargo-raze` to their WORKSPACE file and run it as a tool built purely in Bazel for environments that do not have Cargo installed on the host.

### CODE REVIEW

**Comment by UebelAndre** [comment]
@acmcarther This PR is also blocked behind https://github.com/google/cargo-raze/pull/379 and a release, after which the comments will need to be updated in this PR.

**Comment by UebelAndre** [comment]
@acmcarther Ok, updated documentation, thanks for the release! 😄

File: README.md [hunk 1]
```diff
@@ -225,6 +225,41 @@ $ cargo raze
...
+load("@cargo_raze//:transitive_deps.bzl", "cargo_raze_transitive_deps")
+
+cargo_raze_transitive_deps()
+```
+
+With this in place, users can run the `@cargo_raze//:raze` target to generate new BUILD
+files. eg:
+
+```bash
+bazel run @cargo_raze//:raze --manifest-path=$(pwd)/Cargo.toml
```
Comment (acmcarther) [inline README.md:260]: Doesn't this need to be 
```
bazel run @cargo_raze//:raze -- --manifest-path=$(pwd)/Cargo.toml
```
?

Comment (UebelAndre) [reply README.md:260]: Yes! Thank you!

**Comment by UebelAndre** [comment]
@acmcarther Can you make a patch release for this change? 🙏

**Comment by acmcarther** [comment]
Why?

**Comment by acmcarther** [comment]
(I don't feel strongly, but this change is just documentation, so not clear to me why it would have a release)

**Comment by UebelAndre** [comment]
It's for the fix to `//:raze`. But this communicates the this change so thought it'd be nice after this one rather than before.

**Comment by UebelAndre** [comment]
But also not a big deal if you don't think that warrants a patch

**Comment by UebelAndre** [comment]
On second thought, this should wait for some release so the documentation can be 100% correct.

**Comment by tschuett** [comment]
NFO: Elapsed time: 336.737s, Critical Path: 301.52s
INFO: 328 processes: 68 internal, 260 darwin-sandbox.
INFO: Build completed successfully, 328 total actions
INFO: Build completed successfully, 328 total actions
Error: Raze failed to render with cause: "Failed to find expected vendored crates in "/Users/xxxx/Work/cache/cargo/vendor": ["adler32-1.2.0", "anyhow-1.0.38", "async-stream-0.3.0", "async-stream-impl-0.3.0", "async-trait-0.1.42"] and 195 others. Did you forget to run cargo vendor?"`
`

I tried the documented way to build the dependencies and got this error.

**Comment by UebelAndre** [comment]
> It would help to mention that you still have to run cargo version ....

Do you mean `cargo vendor`? I've added a note but all this section is supposed to communicate is the `cargo-raze` tool can be run through Bazel. Perhaps this warrants a feature request for having `cargo-raze` automatically vendor your source code? Sounds reasonable behind a command line argument.

**Comment by tschuett** [comment]
Sorry. I meant cargo vendor. I misread this feature as an alternative to the vendored workflow.

**Comment by UebelAndre** [comment]
@acmcarther Pinging this ticket again. Can you make a patch release for the API fixes so I can update the docs and get this merged?

**Comment by acmcarther** [comment]
LGTM

**Pull request merged**